opal/util/Makefile.am

```diff
@@ -65,7 +65,9 @@ headers = \
         strncpy.h \
         sys_limits.h \
         timings.h \
-        uri.h
+        uri.h \
+ 	info_subscriber.h \
```

Comment:
Ok, I'm surprised there are not lots of these (maybe there are).   I wasn't super careful because I thought that the commit hook script checked for this.   I don't think my commit hook script is setup correctly though because the copyrights did not get updated either (as you noted in another comment).   I'm looking into that.